cmd/context/create.go

```diff
@@ -108,15 +109,20 @@ func (c *ContextUse) UseContext(ctx context.Context, ctxOptions *ContextOptions)
 	}
 
 	if !ctxOptions.IsOkteto {
-		if !isValidCluster(ctxOptions.Context) {
+		isNewUrl := isNewUrl(ctxOptions.Context)
+		if !isValidCluster(ctxOptions.Context) && !isNewUrl {
 			return errors.UserError{E: fmt.Errorf("invalid okteto context '%s'", ctxOptions.Context),
 				Hint: "Please run 'okteto context' to select one context"}
 		}
-
-		transformedCtx := okteto.K8sContextToOktetoUrl(ctx, ctxOptions.Context, ctxOptions.Namespace, c.k8sClientProvider)
-		if transformedCtx != ctxOptions.Context {
-			ctxOptions.Context = transformedCtx
+		if isNewUrl {
```

Comment:
Using the same name as the function in the same package is a bit odd. Is this always non-nil and therefore true because it's a function here or is this the return value and the function is no longer usable?